cmd/internal/cli/remote.go

```diff
@@ -230,6 +246,52 @@ func setKeyserver(_ *cobra.Command, _ []string) {
 	}
 }
 
+var RemoteGetLoginPasswordCmd = &cobra.Command{
+	DisableFlagsInUseLine: true,
+
+	Use:     docs.RemoteGetLoginPasswordUse,
+	Short:   docs.RemoteGetLoginPasswordShort,
+	Long:    docs.RemoteGetLoginPasswordLong,
+	Example: docs.RemoteGetLoginPasswordExample,
+
+	Run: func(cmd *cobra.Command, args []string) {
+		path := "/v1/rbac/users/current"
+		baseURL := "https://library.sylabs.io"
+		if libraryURI != "" {
+			baseURL = libraryURI
+		}
+		endPoint := baseURL + path
+		defaultConfigURI := ""
+
+		config, err := getLibraryClientConfig(defaultConfigURI)
+		if err != nil {
+			sylog.Fatalf("config err: %v", err)
+		}
+
+		req, err := http.NewRequest("GET", endPoint, nil)
+		if err != nil {
+			sylog.Fatalf("request err: %v", err)
+		}
+		req.Header.Set("Authorization", fmt.Sprintf("Bearer %v", config.AuthToken))
```

Comment:
Maybe add some more error checking around the auth token and status code, I was playing around with this pr and got a 401 from the library and singularity just exited with 255 with no info about why.